src/database/standalone-sqlite.ts

```
@@ -3812,6 +3832,13 @@ if (!isMainThread) {
           const failedBundleIds = worker.getFailedBundleIds(args[0]);
           parentPort?.postMessage(failedBundleIds);
           break;
+        case 'getLargeFailedBundleIds':
+          const largeFailedBundleIds = worker.getLargeFailedBundleIds(
+            args[0],
+            args[1],
+          );
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)